editor/editor_node.cpp

```diff
@@ -6967,7 +6967,15 @@ EditorNode::EditorNode() {
 		String rendering_method = renderers[i];
 
 		// Add the renderers name to the UI.
-		renderer->add_item(rendering_method);
+		if (rendering_method == "forward_plus") {
+			renderer->add_item(TTR("Forward+"));
+		}
+		if (rendering_method == "mobile") {
+			renderer->add_item(TTR("Mobile"));
+		}
+		if (rendering_method == "gl_compatibility") {
+			renderer->add_item(TTR("Compatibility"));
+		}
```

Comment:
I tested out locally and it looks like the color of the text wasn't changing from white. It looks like you will have to update the string comparisons in _update_renderer_color as they are relying on the text of the label matching the rendering_method names

Reply:
You're correct @clayjohn  !
I've got it fixed locally, but before pushing, I just wanted to make sure that the colors are really needed.
Forward+ an mobile are red, which gives the impression that something is wrong.

Forward & mobile now look like this : 

![2022-12-23_23-57_1](https://user-images.githubusercontent.com/3624853/209411861-354f2190-8977-403b-80d8-23c3e28fa904.png)

Compat : 

![2022-12-23_23-57](https://user-images.githubusercontent.com/3624853/209411878-ec897fe5-9258-452d-915b-9a86ce4dae84.png)

Reply:
The previous colors were most likely placeholders. I'd probably use the following colors:

- Forward+: Use a new green color, not too saturated
- Mobile: Reuse the 3.x GLES3 color
- Compatibility: Reuse the 3.x GLES2 color

Reply:
I agree the red is over the top. Calinou's suggestion sounds good!